demo/twenty_news/build.gradle

```diff
@@ -17,15 +17,10 @@ dependencies {
   compile 'org.slf4j:slf4j-simple:1.7.7'
 }
 
-compileScala {
```

Comment:
The `fork` option has been removed in Gradle 3+ and is causing error in my local environment. https://docs.gradle.org/current/release-notes#ant-based-scala-compiler-has-been-removed